akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -24,6 +26,7 @@ private[stream] object GraphInterpreter {
   final val NoEvent = -1
   final val Boundary = -1
 
+  final val Initialized = 0
```

Comment:
actually all ports should be initialized to `InReady` so this is more like `Uninitialized`. Btw, do you use this value somewhere?